man/memkind.3

```diff
@@ -764,6 +773,12 @@ the request, the allocation will fall back on other memory NUMA nodes.
 .BR Note:
 Only one node ID could be marked as preferred one.
 .TP
+.B MEMKIND_DAX_KMEM_INTERLEAVE
```

Comment:
I suggest resigning from this kind, as we have no direct requests for it.

Reply:
Done.